libs/gi/db/src/Database/DataManagers/OptConfigDataManager.ts

```diff
@@ -85,6 +85,8 @@ export interface OptConfig {
   compareBuild: boolean
   levelLow: number
   levelHigh: number
+  useLoadoutExclusion: boolean // teammates loadout exclusion flag
```

Comment:
I would rename this to `useTeammateBuild`. We want to exclude only the active build on teammates, not their entire loadout.